dask_cuda/cuda.yaml

```diff
@@ -0,0 +1,8 @@
+cuda:
```

Comment:
I have no strong opinions/objections. I think we can keep it in the same namespace but need a different file to prevent dask-cuda and dask-distributed from overwriting one another's configurations. Would you like me to change it to `distributed` namespace?